README.md

```diff
@@ -1,3 +1,6 @@
 # :cat: vs :dog:
 
 The problem: given a collection of labelled cats and dogs pictures, train a neural network to classify them.
+
+### Technologies
```

Comment:
This Heading should be level 2 (only 2 # ).

Reply:
change applied